pkg/controller/controller.go

```diff
@@ -235,7 +239,8 @@ func NewCSIProvisioner(client kubernetes.Interface,
 	pluginCapabilities connection.PluginCapabilitySet,
 	controllerCapabilities connection.ControllerCapabilitySet,
 	supportsMigrationFromInTreePluginName string,
-	strictTopology bool) controller.Provisioner {
+	strictTopology bool,
+	informer cache.SharedIndexInformer) controller.Provisioner {
```

Comment:
instead of passing in the entire informer to the provisioner, how about only passing in the Lister(), since all we need is Get() and List()

Reply:
I'll change to `informer.GetStore()`